createdisk-library.sh

```diff
@@ -111,16 +111,24 @@ function copy_additional_files {
     update_json_description $srcDir $destDir $podmanVersion
 }
 
-function prepare_hyperV() {
+function install_packages() {
     local vm_ip=$1
     # Install the hyperV rpms to VM
     ${SSH} core@${vm_ip} -- 'sudo sed -i -z s/enabled=0/enabled=1/ /etc/yum.repos.d/fedora.repo'
     ${SSH} core@${vm_ip} -- 'sudo sed -i -z s/enabled=0/enabled=1/ /etc/yum.repos.d/fedora-updates.repo'
-    ${SSH} core@${vm_ip} -- 'sudo rpm-ostree install --allow-inactive hyperv-daemons'
+    if [ -n "${SNC_GENERATE_WINDOWS_BUNDLE}" ]; then
+        prepare_hyperV ${vm_ip}
+    fi
     ${SSH} core@${vm_ip} -- 'sudo sed -i -z s/enabled=1/enabled=0/ /etc/yum.repos.d/fedora.repo'
     ${SSH} core@${vm_ip} -- 'sudo sed -i -z s/enabled=1/enabled=0/ /etc/yum.repos.d/fedora-updates.repo'
     ${SSH} core@${vm_ip} -- 'sudo rpm-ostree cleanup --base'
     ${SSH} core@${vm_ip} -- 'sudo rpm-ostree cleanup --repomd'
+}
+
+function prepare_hyperV() {
+    local vm_ip=$1
+    # Install the hyperV rpms to VM
```

Comment:
I'd remove this comment as well, `ssh $vm_ip rpm-ostree install hyperv-daemons` is self explanatory